src/main/java/org/iac2/service/architecturereconstruction/common/exception/NameSpaceNotFoundException.java

```diff
@@ -0,0 +1,19 @@
+package org.iac2.service.architecturereconstruction.common.exception;
+
+public class NameSpaceNotFoundException extends RuntimeException {
```

Comment:
Please make this exception inherit from one of the exceptions defined in `org.iac2.common.exception`